libbindgen/src/clang.rs

```diff
@@ -190,6 +190,10 @@ impl Cursor {
         unsafe { clang_getCursorKind(self.x) }
     }
 
+    pub fn is_definition(&self) -> bool {
+        unsafe { clang_isCursorDefinition(self.x) == 0 }
```

Comment:
This is returning true if the cursor is _not_ a definition, which causes a lot of confusion. Can you change this check to `!= 0`, and invert the conditional it in other places?

Also, this is going to need a line of documentation or this won't pass CI.